inc/class_captcha.php

```diff
@@ -33,6 +33,13 @@
  *
  */
 
+define('DEFAULT_CAPTCHA', 1);
+define('NOCAPTCHA_RECAPTCHA', 4);
+define('RECAPTCHA_INVISIBLE', 5);
+define('HCAPTCHA', 6);
+define('HCAPTCHA_INVISIBLE', 7);
+define('RECAPTCHA_V3', 8);
+
```

Comment:
Personally I think these would be best placed as constants within the `captcha` class to emphasise the need to `require` this file.

Thoughts @euantorano?

Reply:
Agreed, that also fits more with the code standards we are going for with 1.9 and ahead where one file does one thing (e.g. define a class).